### PR TITLE
Abb217 juste

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record.

### CODE REVIEW

File: ParisBNF/abb/BNFabb216.xml [hunk 1]
```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BNFabb216" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>A short grammair of Ethiopic, A treatise on the Ethiopian qǝne-poetry by Giusto da Urbino</title>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb216.xml:8]: grammar

File: ParisBNF/abb/BNFabb216.xml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                  <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10087255m">BnF Éthiopien d'Abbadie 216</idno>
+
+                    </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+
+                  <msItem xml:id="ms_i1">
+                            <locus from="1r" to="115v"/>
+                            <title type="complete" ref="STU0006Abrege">A short grammare of Ethiopic written by Giusto da Urbino in 1850-1854</title>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb216.xml:39]: grammar

**Review by thea-m — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb216.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb216.xml:8]: ok.

File: ParisBNF/abb/BNFabb216.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb216.xml:39]: Ok, thanks!

**Review by eu-genia — APPROVED** [review]

**Pull request merged**